FluentUI/src/main/java/com/microsoft/fluentui/contextualcommandbar/CommandItem.kt

```diff
@@ -0,0 +1,15 @@
+package com.microsoft.fluentui.contextualcommandbar
+
+import android.support.annotation.DrawableRes
+
+interface CommandItem {
```

Comment:
should we have doc comments for the classes/methods? it looks like that is the norm for the library code...